source/codegen/metadata/nidmm/functions.py

```diff
@@ -696,7 +696,8 @@
         {
             "name": "sampleTriggerSlope",
             "direction": "in",
-            "type": "ViInt32"
+            "type": "ViInt32",
+            "enum": "Slopes"
```

Comment:
Question: is this information encoded in the base hapigen metadata somewhere?

Reply:
that one 3 of the enums that are in hapigen were combined as they have same values, but now that change is reverted to keep consistency with the enums.hapigen